app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -353,41 +353,91 @@ class PluginApi {
    * ```
    **/
   addPosterIcon(cb) {
+    this.addPosterIcons(cb);
```

Comment:
I think we should change the description for `api.addPosterIcon` to reflect the changes. Maybe just state that it is an alias of `addPosterIcons`?

Reply:
Sounds good